### ISSUE TITLE
Advise for Neural Network 

### ISSUE THREAD

**Issue by Nate75015** [jpmml/sklearn2pmml#39]
Good evening Mr Ruusman,

I am coming to search advise from you. Indeed in this moment my goal is to export a Neural Network in PMML. My code is really basic. But when I try to execute it with the Standalone version there is a little problem ... The target and the associate probability are null.

You can see it. Could you explain to me what is wrong ?

Thanks,

![capture d ecran 2017-06-05 a 18 27 04](https://cloud.githubusercontent.com/assets/27303125/26793012/abacd762-4a1c-11e7-9b62-7caf77e5c473.png)

**Comment by vruusmann** [comment]
> You can see it. Could you explain to me what is wrong?

You should show me your Python code instead.

It seems to be the case that you're trying to build a classification-type neural network model for the Wine Quality dataset. However, the target column - "quality" - is a continuous variable (double in range from 3 to 8), not a categorical variable (string one of "3", "4", "5", "6", "7", "8"). So, you should be approaching it as a regression problem, not as a classification problem.

In other words, replace `sklearn.MLPClassifier` with `sklearn.MLPRegressor`, and re-run the experiment

**Comment by Nate75015** [comment]
Hi vruusmann,

I try what you told me to do ... But it seems to be have another problem. Does it possible to let me an adress where I could put my code ? 

Thanks to you !
